### ISSUE TITLE
Finetune FaceNet-Pytorch on CelebA dataset

### ISSUE THREAD

**Issue by jannikjw** [jannikjw/one-shot-face-recognition#20]
Goal: Record accuracy on CelebA-one-shot dataset -> see if improved
